topics/master/secrets.txt

```diff
@@ -0,0 +1,71 @@
+Secrets Configuration
+---------------------------------------------------------------------------------------------------
+Secrets configuration functionality is provided so properties containing sensitive information can be stored and managed without
+checking into source control.
+
+*** Setting up the standard approach to use secrets functionality ***
```

Comment:
Recommend wording: "Note: while Hodor's Secrets Configuration facilities can accommodate any number of secret files, the standard approach recommends you use three of them, each fulfilling..."